### PR TITLE
Adding db_schema and whitelisting json along with structure format code

### PR DESCRIPTION
**Pull request by sofiya02**

### CODE REVIEW

File: etc/db_schema.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+           <column xsi:type="varchar" name="cancel_by" nullable="false" length="30"  comment="Razorpay Subscription cancel by user or admin"/>
+            <column xsi:type="int" name="total_count" nullable="false"  default="0" comment="Total subscription count to be charged"/>
+             <column xsi:type="int" name="paid_count" nullable="false" default="0" comment="Total subscription paid count"/>
+         <column xsi:type="int" name="remaining_count" nullable="false" default="0" comment="Subscription remaining to be charged"/>
+  <column xsi:type="int" name="auth_attempts" nullable="false" default="0" comment="The number of times that the charge for the current billing cycle has been attempted on the card."/>
+     <column xsi:type="timestamp" name="start_at" on_update="false" nullable="true" default="CURRENT_TIMESTAMP" comment="Subscription start date"/>
+ <column xsi:type="timestamp" name="end_at" on_update="false" nullable="true" default="CURRENT_TIMESTAMP" comment="Subscription end date"/>
+ <column xsi:type="timestamp" name="subscription_created_at" on_update="false" nullable="true" default="CURRENT_TIMESTAMP" comment="Subscription Created at"/>
```
Comment (HancilSequeira) [inline etc/db_schema.xml:55]: Format this file

**Review by HancilSequeira — CHANGES_REQUESTED** [review]
Remove .DS_store file

File: etc/db_schema.xml (hunk 1 above)
Comment (sofiya02) [reply etc/db_schema.xml]: done

**Comment by sofiya02** [comment]
don



> Remove .DS_store file

done

File: Setup/UpgradeSchema.php [hunk 2]
```diff
@@ -11,13 +14,22 @@
 use Razorpay\Subscription\Model\ResourceModel\Subscriptions;
 use Razorpay\Subscription\Model\ResourceModel\SubscriptionsOrderMapping;
 /**
- * @UpgradeSchema to update the current db with subscription 
+ * @UpgradeSchema 
+ * update the current db with subscription
```
Comment (HancilSequeira) [inline Setup/UpgradeSchema.php:18]: Did you tested without this `Setup/UpgradeSchema.php`

**Review by HancilSequeira — CHANGES_REQUESTED** [review]

**Review by HancilSequeira — APPROVED** [review]

File: Setup/UpgradeSchema.php (hunk 2 above)
Comment (sofiya02) [reply Setup/UpgradeSchema.php:18]: yes

**Pull request merged**